internal/pkg/template/templates/environment/cf.yml

```diff
@@ -581,7 +581,9 @@ Resources:
 {{include "lambdas" . | indent 2}}
 {{include "custom-resources" . | indent 2}}
 {{- end}}
+{{- if not .VPCConfig.Imported}}
```

Comment:
Is this impossible to support or do we just need time to decide whether to support it in the future?

Reply:
i think it's similar to the NAT gateway handling - we _could_ support it, but since we don't know if customers have already deployed something in the imported VPC, we'll avoid creating one and have them be in charge of those environment resources that cost $.